adapt translation logic

OCurdy opened the pull request:
Adapt `mco:otherConstraints` to have a drop-down list to help the user to select options from the codelist **`access_use_condition`**. We have to add also the file that my colleague (@taniahumar) is finishing to write in 
geocat-wo/web/src/main/webapp/WEB-INF/data/config/codelist/external/thesauri/theme/access_use_conditions.rdf

cmangeat commented:
The change should please take place in this file: https://github.com/geoadmin/geocat-wo/blob/master/web/src/main/webapp/WEB-INF/data/config/schema_plugins/iso19115-3.2018.che/layout/config-editor.xml (the one overriding editor config for swiss topo).

Commit:
Add Swiss constraints choices and update protocol dropdown defaults

- Add access/use conditions dropdown options for mco:otherConstraints
- Update default protocol selection in all language files (eng/fre/ger/ita)
- Remove outdated WWW:LINK-1.0-http--link as default protocol

OCurdy commented:
I have add the list of protocols we had in prod with iso19139.che. I just commented the initial protocols element. Please @cmangeat can you just check if everything is alright for you an merge if it is?
